README.md

```diff
@@ -4,42 +4,31 @@
 
-The username that should be supplied at login is `ros`.
+The `ros2_ws` image provides a ROS2 workspace with pre-installed ros2-control packages.
 
-### ROS workspace with pre-installed control libraries
+### Deprecated images
 
-The ros_control_libraries image provides the same ROS workspace as ros_ws but
-comes with the pre-installed [control libraries](https://github.com/aica-technology/control-libraries).
+1. ROS workspace (noetic): the ros_ws image provides a ROS workspace (noetic). Build it by running the
+   [build script](ros_ws/build.sh) while in the [ros_ws](ros_ws) directory. The username that should be supplied at
+   login is `ros`.
 
-### ROS2 workspace
+2. ROS workspace with pre-installed control libraries: the ros_control_libraries image provides the same ROS workspace
+   as ros_ws but comes with the pre-installed [control libraries](https://github.com/aica-technology/control-libraries).
 
-Similar to the ROS workspace, the ros2_ws image provides a ROS2 workspace.
-Build it by running the [build script](ros2_ws/build.sh) while
-in the [ros2_ws](ros2_ws) directory. Modify the ROS2 distribution
-in the script as desired.
+3. ROS2 workspace with pre-installed control libraries: the ros2_control_libraries image provides the a ROS2 workspace
+   and comes with the pre-installed [control libraries](https://github.com/aica-technology/control-libraries).
 
-The username that should be supplied at login is `ros2`.
+4. ROS2 workspace with pre-installed control libraries and modulo: the ros2_modulo image provides a ROS2 workspace with
+   [modulo](https://github.com/aica-technology/modulo) already built and comes with pre-installed
+   [control libraries](https://github.com/aica-technology/control-libraries).
 
-### ROS2 workspace with pre-installed control libraries
-
-The ros2_control_libraries image provides the same ROS2 workspace as ros2_ws but
-comes with the pre-installed [control libraries](https://github.com/aica-technology/control-libraries).
-
-### ROS2 workspace with pre-installed control libraries and modulo
-
-The ros2_modulo image provides a ROS2 workspace with [modulo](https://github.com/aica-technology/modulo)
-already built and comes with pre-installed [control libraries](https://github.com/aica-technology/control-libraries).
-
-### ROS2 workspace with pre-installed control libraries, modulo, and ROS2 control
-
-The ros2_modulo_control image provides a ROS2 workspace with [modulo](https://github.com/aica-technology/modulo)
-already built and comes with pre-installed [control libraries](https://github.com/aica-technology/control-libraries)
-and ROS2 control packages.
+5. ROS2 workspace with pre-installed control libraries, modulo, and ROS2 control: the ros2_modulo_control image provides
+   a ROS2 workspace with [modulo](https://github.com/aica-technology/modulo) already built and comes with pre-installed
+   [control libraries](https://github.com/aica-technology/control-libraries) and ROS2 control packages.
 
 ## Scripts
```

Comment:
Do we want to stop supporting scripts at some point in this initiative? (i.e. put them under the ## deprecated section)

Reply:
They still work though, we just don't advertise them anymore. I'd keep them there for now, it can be a good point of reference.